PWGEM/Dilepton/Tasks/emEfficiencyEE.cxx

```diff
@@ -1188,7 +1195,15 @@ struct AnalysisSameEventPairing {
 
       int isig = 0;
       for (auto sig = fMCSignals.begin(); sig != fMCSignals.end(); sig++, isig++) {
-        if ((*sig).CheckSignal(true, groupedMCTracks, t1, t2)) {
+        bool checked = false;
+        if constexpr (soa::is_soa_filtered_v<TTracksMC>) {
+          auto t1_raw = groupedMCTracks.rawIteratorAt(t1.globalIndex());
+          auto t2_raw = groupedMCTracks.rawIteratorAt(t2.globalIndex());
+          checked = (*sig).CheckSignal(false, t1_raw, t2_raw);
```

Comment:
same here. Use
`checked = (*sig).CheckSignal(true, t1_raw, t2_raw);`
and in line 1204.